contracts/test/DeployableERC20Detailed.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.0;
+
+// NOTE:
+//  This file's purpose is to have an ERC20Detailed contract, which is not
```

Comment:
I could not find a better solution that having this separate test contract :(

Reply:
nit: maybe mention that this is needed for the tests.

Reply:
I am not sure what the need for this is unless you don't have a `Dependencies.sol`. For example, in `dex-liquidity-provision` we use the same contract and include an import statement in ...

https://github.com/gnosis/dex-liquidity-provision/blob/b35a36d2054f8a78ebffe0fc70dd6e79eaa75e50/contracts/Dependencies.sol#L1-L9